docs/guide/dev/installation/README.md

```diff
@@ -80,6 +93,24 @@ When you are done, open your test page in the browser.
 
 Instead of downloading CKEditor to your server and hosting it you can also use the CDN version. Go to the [official CKEditor CDN](http://cdn.ckeditor.com/) page for more details.
 
+## CKEditor Framework Integrations:
+
+CKEditor 4 supported frameworks:
+
+### [CKEditor 4 React Integration](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_react.html):
```

Comment:
My proposal is to make it a regular title (no link):

```
### CKEditor 4 React Integration
```

And then go with something like:

Official CKEditor 4 React Integration package/component (not sure how it should be called) can be installed via NPM:

```bash
npm install --save ckeditor4-react
```

By default it will automatically fetch latest CKEditor 4 build via CDN. Check the integration guides on how it can be changed and how to configure the component to fit you needs.

---

The same for Angular. And for Vue, you can also include the header, leave the description as is but link also to docs Vue section (you need https://github.com/ckeditor/ckeditor-docs/pull/247 for that).